helm/db-controller/values.yaml

```
@@ -184,3 +184,5 @@ controllerConfig:
     ib_env: "{{ .Values.env }}"
     ib_lifecycle: "{{ .Values.lifecycle }}"
 
+dashboard:
+  enabled: true
```

```suggestion
  enabled: false
```
Don't want to add a dependency if it's not needed